src/main/java/com/github/princesslana/smalld/HttpClient.java

```diff
@@ -82,17 +84,30 @@ public WebSocket newWebSocket(Request request, WebSocketListener listener) {
    * Sends a request build with the builder to the given path. The path is relative to the base url
    * that is retrieved from the {@link Config} provided when the {@code HttpClient} was initialized.
    *
+   * <p>This send request is invoked when the request sent contains query parameters.
```

Comment:
Doc nits:
"this" send request? there's only one

"This get request", nothing specific to Get about this method. I'd also use where the value is a String, rather than Object is a String.

Reply:
ah you're right, I'll fix the docs, I'll look over all of em, I messed em up by copy pastaing